app.rb

```diff
@@ -0,0 +1,24 @@
+# main class rack app.
+class App
+  def call(env)
+    req = Rack::Request.new(env)
+    case req.path_info
+    when /time/
+      input = req.params['format'].split(',')
+      status_code, body = Devider.new(input).handle_time
+      make_response(status_code, plain_headers, body)
+    else
+      make_response(404, plain_headers, "Page not found\n")
+    end
+  end
+
+  private
+
+  def plain_headers
+    { 'Content-Type' => 'text/plain' }
+  end
+
+  def make_response(status_code, headers, body)
```

Comment:
Можно использовать дефолтное значения, это сократит вызовы
```suggestion
  def make_response(status_code, headers = plain_headers, body)
```